test/e2e/frontend/cypress/tests/team/billing.js

```diff
@@ -51,4 +51,93 @@ describe('FlowForge - Team Billing', () => {
             })
         })
     })
+
+    describe('Cancelled subscriptions', () => {
+        it('redirects regular users to the billing page', () => {
+            cy.login('bob', 'bbPassword')
+
+            cy.intercept('GET', '/api/v1/teams/*', (req) => req.reply(res => {
+                res.body = {
+                    ...res.body,
+                    ...{
+                        billing: {
+                            active: false,
+                            canceled: true,
+                            customer: 'cus_Q7HgF3LrYhO3r2',
+                            pastDue: false,
+                            subscription: 'sub_1PH36qJ6VWAujNoL8DX1oE7V',
+                            unmanaged: false
+                        }
+                    }
+                }
+                return res
+            })).as('getTeams')
+
+            cy.visit('/')
+
+            cy.wait('@getTeams')
+
+            cy.url().should('include', '/team/ateam/billing')
+
+            cy.get('[data-nav="team-applications"').click()
+            cy.url().should('include', '/team/ateam/billing')
+            cy.get('[data-nav="team-instances"').click()
+            cy.url().should('include', '/team/ateam/billing')
+            cy.get('[data-nav="team-devices"').click()
+            cy.url().should('include', '/team/ateam/billing')
+            cy.get('[data-nav="shared-library"').click()
+            cy.url().should('include', '/team/ateam/billing')
+            cy.get('[data-nav="team-members"').click()
+            cy.url().should('include', '/team/ateam/billing')
+            cy.get('[data-nav="team-audit"').click()
+            cy.url().should('include', '/team/ateam/billing')
+            cy.get('[data-nav="team-billing"').click()
+            cy.url().should('include', '/team/ateam/billing')
+
+            cy.get('[data-nav="team-settings"').click()
+            cy.url().should('include', '/team/ateam/settings/general')
+        })
+
+        it('allows admins to navigate the team', () => {
+            cy.login('alice', 'aaPassword')
+
+            cy.intercept('GET', '/api/v1/teams/*', (req) => req.reply(res => {
+                res.body = {
+                    ...res.body,
+                    ...{
+                        billing: {
+                            active: false,
+                            canceled: true,
+                            customer: 'cus_Q7HgF3LrYhO3r2',
+                            pastDue: false,
+                            subscription: 'sub_1PH36qJ6VWAujNoL8DX1oE7V',
+                            unmanaged: false
+                        }
```

Comment:
probably should make this a common variable with the above test to ensure the same values and de-risk regressions

Reply:
we only need the active and canceled keys in there, the rest is leftover copied code, I'll remove it